### DISCUSSION TITLE
Schedule running 2 times on IST and UTC, when configured to run in IST only

### DISCUSSION THREAD

**Discussion by ajayaghosh** [quartznet/quartznet#1689] [Q&A]
Hi, I'm facing an issue in running the schedule using quartz,
I have scheduled  a process to run at 4:04 AM IST, the scheduler runs at that time as well as at 4:04 AM UTC. Runs 2 times!!
hosted in AWS->EC2->linux->apache
I'm using dotnet core 6 and below is the configuration snippet:

services.AddQuartz(q =>
            {
                // Use a Scoped container to create jobs. I'll touch on this later
                q.UseMicrosoftDependencyInjectionScopedJobFactory();
                var jobKey = new JobKey("MyCronJob");

                // Register the job with the DI container
                q.AddJob<MyCronJob>(opts => opts.WithIdentity(jobKey));

                // Create a trigger for the job
                q.AddTrigger(opts => opts
                    .ForJob(jobKey) // link to the HelloWorldJob
                    .WithIdentity("HelloWorldJob-trigger") // give the trigger a unique name
                    .WithCronSchedule("0 4 4 * * ? "));
            });
  
  What is the reason behind 2 times running of the schedule? i want it to run only in IST.
  
  Thanks in Advance.

**Comment by lahma** [comment]
Please note that if you don't explicitly define time zone in your configuration, server's local time zone will be used. For cloud services like Azure and AWS, this usually is configured to be UTC.
